dbt/models/ccbor/schema.yml

```diff
@@ -0,0 +1,70 @@
+sources:
+  - name: ccbor
+    tags:
+      - load_manual
+    tables:
+      - name: appeals
+        description: '{{ doc("table_appeals") }}'
+        columns:
+          - name: appeal_id
+            description: Appeal ID. Matches the "Complaint" value on the Board of Review Decision Search.
```

Comment:
**mega nitpick:** We never have period in the descriptions for single-lines, only if there's a line break:

```suggestion
            description: Appeal ID. Matches the "Complaint" value on the Board of Review Decision Search
```

Also applies to the descriptions below.